robusta_krr/core/integrations/prometheus/prometheus_utils.py

```diff
@@ -25,9 +25,10 @@ def generate_prometheus_config(
     url: str, headers: dict[str, str], metrics_service: PrometheusMetricsService
 ) -> PrometheusConfig:
     from .metrics_service.victoria_metrics_service import VictoriaMetricsService
+    from .metrics_service.mimir_metrics_service import MimirMetricsService
 
     baseconfig = {
-        "url": url,
+        "url": url + "/prometheus" if isinstance(metrics_service, MimirMetricsService) else url,
```

Comment:
Also ugly, but it only works when adding this to the URL.

Reply:
https://grafana.com/docs/mimir/latest/configure/configure-the-query-frontend-work-with-prometheus/
I think this can be avoided by setting `api.prometheus_http_prefix` to `''`.